schema-guide.md

```diff
@@ -444,37 +486,49 @@ primary principle, "Importance", when others cite this work.
 
 ## Definitions
 
-TODO explain why we use definitions
+Some values in CFF files are valid in different fields.
+For example, `repository-code`, `url` and `license-url` all take URLs as values.
+
+The schema therefore has [*definitions*](https://json-schema.org/understanding-json-schema/structuring.html#definitions)
+of smaller subschemas, that can be reused in the schema from the respective field, 
+instead of having to duplicate them.
+For example, there is one definition for a valid URL value ([`definitions.url`](#definitionsurl)),
+that is being referenced from `repository-code`, `url` and `license-url`.
+
+**Note:** Definitions are NOT field names, although they may be called similarly.
+Do NOT use definitions or subkeys (like `definitions.alias` or `definitions.entity.alias`)
+when writing a CFF file.  
+To make the definition sections more distinct, their headers are printed in italics, 
+such as *`definitions.alias`*.
```

Comment:
I prefer to not have the italics. For me they raise questions, more than anything. `definitions` are a standard, well documented in any JSONschema related docs, plus the actual word `definitions` is right there in the heading. To top it all off, we created a separate section for them.